### PR TITLE
Add settings view commands

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

Changes are related to #4011, this set ensures that any custom formatter does not need to concern itself with what a group command might be doing alongside help.

### CODE REVIEW

**Comment by Drapersniper** [comment]
@Drapersniper Just a note to self... this will be needed in audio rewrite and audio api.

File: redbot/cogs/economy/economy.py [hunk 1]
```diff
@@ -655,34 +655,39 @@ async def slot_machine(author, channel, bid):
...
-                        slot_time=humanize_number(await conf.SLOT_TIME()),
-                        payday_time=humanize_number(await conf.PAYDAY_TIME()),
-                        payday_amount=humanize_number(await conf.PAYDAY_CREDITS()),
-                        register_amount=humanize_number(await bank.get_default_balance(guild)),
-                        maximum_bal=humanize_number(await bank.get_max_balance(guild)),
-                    )
+        if await bank.is_global():
+            conf = self.config
+        else:
+            conf = self.config.guild()
```
Comment (Jackenmen) [inline redbot/cogs/economy/economy.py]:
```suggestion
            conf = self.config.guild(guild)
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]
Looks good, just one small bug with `[p]economyset showsettings` and there's some conflict to fix.

**Review by Jackenmen — APPROVED** [review]
Looks good to me.

**Pull request merged**